dotcom-rendering/src/web/components/SignInGateSelector.importable.tsx

```diff
@@ -192,6 +214,12 @@ export const SignInGateSelector = ({
 				})
 				.then(setCanShowGate);
 		}
+		// eslint-disable-next-line @typescript-eslint/no-floating-promises
+		getSwitches().then((switches) => {
```

Comment:
I think you can put a `void` before the promises so you don't need to disable the eslint check:
```suggestion
		void getSwitches().then((switches) => {
```

Reply:
Thanks, I'll add that